packages/cloud-cognitive/src/components/Datagrid/Datagrid/DatagridContent.js

```diff
@@ -89,7 +89,7 @@ export const DatagridContent = ({ datagridState }) => {
         }
         onFocus={withInlineEdit ? () => handleGridFocus(state, dispatch) : null}
       >
-        <DatagridHead {...datagridState} />
+        {!withVirtualScroll ? <DatagridHead {...datagridState} /> : ''}
```

Comment:
Instead of an empty string can we render `null`. Or we can do 
```
 {!withVirtualScroll && <DatagridHead {...datagridState} /> }
```

Reply:
@AlexanderMelox Updated as `null`